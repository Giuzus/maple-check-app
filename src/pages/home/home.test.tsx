import { describe, expect, test } from "vitest";
import { render } from "@testing-library/react";
import Home from "./home.page";
import { MemoryRouter } from "react-router-dom";

describe("Home page", () => {
  test("should match snapshot", () => {
    //Arrange
    const page = render(
      <MemoryRouter>
        <Home />
      </MemoryRouter>
    );

    //Assert
    expect(page.baseElement).toMatchSnapshot();
  });
});
