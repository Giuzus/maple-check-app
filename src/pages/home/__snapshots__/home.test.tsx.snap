// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Home page > should match snapshot 1`] = `
<body>
  <div>
    <section>
      <div
        class="join"
      >
        <input
          class="input input-bordered join-item"
          placeholder="Type here"
          type="text"
          value=""
        />
        <button
          class="btn join-item"
        >
          Add
        </button>
      </div>
      <ul />
    </section>
  </div>
</body>
`;
